src/main/java/net/sf/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -432,4 +434,38 @@ public static void openBrowser(String url) throws IOException {
         ExternalFileType fileType = ExternalFileTypes.getInstance().getExternalFileTypeByExt("html");
         openExternalFilePlatformIndependent(fileType, url);
     }
+
+    public static void openConsole(JabRefFrame frame) throws IOException {
```

Comment:
Or maybe pass the database context. This would correspond more to the semantic `open console for this database`